spec/spec_helper.rb

```diff
@@ -59,6 +59,10 @@ def photo_fixture_name
   @photo_fixture_name = File.join(File.dirname(__FILE__), "fixtures", "button.png")
 end
 
+def jwks_file_path
+  @jwks_file = "../../spec/fixtures/jwks.json"
```

Comment:
While Postgres text datatype is unlimited, MySQL's is limited to 64 Kilobyte and it offers MEDIUMTEXT and LONGTEXT for 16 Megabyte and 4 Gigabyte respectively.

But yeah.